Modules/commandcontext.py

```diff
@@ -0,0 +1,104 @@
+from functools import reduce
+from operator import xor
+from typing import List, Union
+
+import pydle
+
+from Modules.user import User
+
+
+class CommandContext(object):
```

Comment:
We will be throwing these objects around a lot. Perhaps `Context` would be much shorter and quicker to use while not sacrificing conciseness?